File: src/plugin/utils/VPNApiResponse.java
package plugin.utils;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class VPNApiResponse {
    public boolean anon;
    public String status;
    public String ip;
    public String isp;
    public String message;
}

File: src/main/kotlin/plugin/model/VPNApiResponse.kt
package plugin.model

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
class VPNApiResponse {
    var anon: Boolean = false
    var status: String? = null
    var ip: String? = null
    var isp: String? = null
    var message: String? = null
}